crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -564,6 +671,8 @@ struct ActiveFilter<T> {
     last_poll_timestamp: Instant,
     /// What kind of filter it is.
     kind: FilterKind<T>,
+    /// reorg blocks that are relevant to this filter
+    reorg_blocks: Arc<RwLock<HashMap<B256, BlockNumber>>>,
```

Comment:
this type is never clone, so I don't think we need the Arc here because the filters are always accessed as ref.

Or I guess we need this because we clone this, right?